api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -67,6 +69,10 @@
   private static final String RANDOM_CHARS = "abcdefghijklmnopqrstuvwxyz";
   private static final int NUM_RANDOM_CHARS = 20;
   private static final int MAX_FC_CREATION_ATTEMPT_VALUES = 6;
+  private static final int MAX_NOTEBOOK_SIZE_MB = 100;
```

Comment:
We could make some of these config settings in future in WorkbenchConfig.

Reply:
Yes - if we wind up needing to tune this. I think my default is to start with a constant unless there's high confidence we'll need to dynamically change this, to avoid bloating the config.

Added comment.